packages/dear-imgui/ts/src/lib/components/TradingGuiDemo/CryptoPlots/CryptoLinePlots.tsx

```
@@ -1,15 +1,15 @@
 import React from "react";
 import { useStore } from "../store";
 import { CryptoLinePlot } from "./CryptoLinePlot";
-import { ReactImgui } from "../../ReactImgui";
+import { XFrames } from "../../XFrames";
 
 type Props = {};
 
 export const CryptoLinePlots = ({}: Props) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)